spec/features/bulk_update_metadata_spec.rb

```diff
@@ -28,6 +28,7 @@
     choose 'Spreadsheet input; load into objects'
     fill_in '3. Note', with: note
     click_button 'Submit'
+    sleep 5 # For some reason, somtimes this upload is slow.
```

Comment:
typo 'sometimes'

Reply:
No, nothing appears after the submit and the check below that involves a refresh interrupts the form submission. We're in some odd Capybara / browser twilight zone here. Open to suggestions.

And fixed typo.

Reply:
How about we add this first? https://github.com/sul-dlss/argo/pull/2715

Reply:
With that change, I was able to replace wait.